conf/distro/asteroid.conf

```diff
@@ -41,3 +41,12 @@ PACKAGE_FEED_BASE_PATHS = "ipk"
 PACKAGE_FEED_ARCHS = "all anthias armv7vehf-neon bass core2-32 dory lenok qemux86 sparrow sprat sturgeon swift tetra wren"
 
 SKIP_META_GNOME_SANITY_CHECK = "1"
+
+# - Add nativesdk-qttools-tools to allow for building
+#   Qt5 code with the SDK. Note that the SDK must be build
+#   based on an image that inherits "populate_sdk_qt5".
+# - Add nativesdk-qttools-dev as well, otherwise the CMake
+#   Qt5LinguistTools scripts are not included in an SDK.
+# - Add nativesdk-cmake to enable CMake support in SDKs.
+#   Newer Qt projects use CMake instead of qmake.
+TOOLCHAIN_HOST_TASK:append = " nativesdk-cmake nativesdk-qttools-tools nativesdk-qttools-dev "
```

Comment:
This is one part where I am uncertain. I do think that this belongs in the distro config, but I do not know if this should use `TOOLCHAIN_HOST_TASK ?=` instead of `TOOLCHAIN_HOST_TASK:append =`.

Reply:
This append is correct but belongs to the image where you added the inherit, as we shouldn't force it in all images.

Reply:
@otavio There is `asteroid-image-dev`. You would add both the `inherit` and the `TOOLCHAIN_HOST_TASK:append` lines to that image?

Reply:
Yes as this is related to the image itself. Another option is a bbappend for the Qt SDK packagegroup.